test_email_validation.py

```diff
@@ -0,0 +1,8 @@
+import email_validation
+import unittest
+
+
+class EmailValidationTest(unittest.TestCase):
+    """unit test for EmailValidation"""
+    def test_validate_email(self):
+        self.assertTrue(email, validate_email())
```

Comment:
1) This is invalid: `assertTrue()` accepts only one argument for test.
2) `email` variable doesn't exist in this scope